WordPressKit/WordPressKit/BlogJetpackSettingsServiceRemote.swift

```diff
@@ -99,6 +99,23 @@ public class BlogJetpackSettingsServiceRemote: ServiceRemoteWordPressComREST {
                                  }, failure: {
                                     error, _ in
                                     failure(error)
+                                })
+    }
+
+    /// Disconnects Jetpack from a site
+    ///
+    public func disconnectJetpackFromSite(_ siteID: Int, success: @escaping () -> Void, failure: @escaping (Error) -> Void) {
+        let endpoint = "jetpack-blogs/\(siteID)/mine/delete"
+        let path = self.path(forEndpoint: endpoint, with: .version_1_1)
+
+        wordPressComRestApi.POST(path!,
+                                 parameters: nil,
+                                 success: {
+                                     _, _ in
```

Comment:
Not important, but I think you can just put `_ in` here

Reply:
Great, thanks for letting me know, I'll update that before merging.